deploy/deploy_citus_server.yml

```diff
@@ -74,7 +72,31 @@
       with_items:
         - postgres
 
-
 - hosts: citus_coordinator
+  become: yes
+  become_method: sudo
+  serial: 1
   roles:
     - role: citus_coordinator
+  tasks:
+    - name: Create back up of postgresql.conf
+      copy:
+        src: "{{ pp_datadir }}/postgresql.conf"
+        dest: "/tmp/postgresql.conf"
+        remote_src: yes
```

Comment:
The documentation stated to back up the file, I think mainly if something went wrong in the setup process, so we can easily revert back. I didn't think we needed it long term, so I put it in tmp. Should we keep it around?

Reply:
@rioh Got it.  Not sure if we need to do that, since the file is generated via Ansible anyways?  Not sure if there's anything sensitive in that file, but placing it in /tmp would make it readable by anyone, so we might want to remove this or put it somewhere else (or just set the permissions to 600), up to you!

Reply:
@rioh Could you just set the permissions to 600 for now, to be cautious?  Thanks!